apps/condo/domains/user/schema/SendMessageToSupportService.js

```diff
@@ -59,6 +59,7 @@ const SendMessageToSupportService = new GQLCustomSchema('SendMessageToSupportSer
             schema: 'sendMessageToSupport(data: SendMessageToSupportInput!): SendMessageToSupportOutput',
             resolver: async (parent, args, context) => {
                 const { data } = args
+                //TODO(Kekmus) why we need meta if not use it here
```

Comment:
The `meta` field is good to store some additional data which not used for model searching.